R/splatPop-estimate.R

```diff
@@ -108,7 +108,7 @@ splatPopEstimateEffectSize <- function(params, eqtl) {
 #' @param params SplatPopParams object containing parameters for the
 #'        simulation of the mean expression levels for the population.
 #'        See \code{\link{SplatPopParams}} for details.
-#' @param means data.frame of real gene means across a population, where
+#' @param gene.means data.frame of real gene means across a population, where
```

Comment:
I assume you thought about this but maybe just double check there isn't a name clash with something else already called `gene.means`.

Reply:
gene.means was also used in splat-simulate.R, I've updated the variable to be called emp.gene.means for empirical.